File: Distrito.java
import java.util.ArrayList;
import java.util.List;

public class Distrito {
    
    private int idDistrito;
    private String nombre;
    private String cp;
    private List<Double> indicadores = new ArrayList<>();
    private String color4ejes;
    private double puntuacion4ejes;

    
    public double getCalidadAire() {
        return 8.5; 
    }

    public int getZonasVerdes() {
        return 12; 
    }
}

File: Distrito.js
class Distrito {
    constructor(idDistrito, nombre, cp, puntuacion4ejes) {
        this.idDistrito = idDistrito;
        this.nombre = nombre;
        this.cp = cp;
        this.indicadores = [];
        this.puntuacion4ejes = puntuacion4ejes;
    }

    getCalidadAire() {
        return 8.5; // Valor de prueba
    }

    getZonasVerdes() {
        return 12; // Valor de prueba
    }
}

module.exports = Distrito;